packages/console/src/assets/docs/guides/native-capacitor/README.mdx

```diff
@@ -40,14 +41,14 @@ const logtoClient = new LogtoClient({
 
 <Step title="Implement the sign-in button">
 
-Now, let's configure the redirect URI. The redirect URI is used to redirect the user back to your application after the authentication flow.
 
-Ensure that the URI redirects to the Capacitor app, for example, `com.example.app://callback`. The value may vary depending on your Capacitor app configuration. For more details, see [Capacitor Deep Links](https://capacitorjs.com/docs/guides/deep-links).
+<RedirectUrisNative defaultUri="com.example.app://callback" />
```

Comment:
can you add file paths when available?

Reply:
Synced offline. Capacitor js does not have a sample code, all the code snippets in this guide do not have a file context. Keep it for now.